### ISSUE TITLE
Enhancing with Integrated PDF Viewing

### ISSUE THREAD

**Issue by biowilli** [krille-chan/fluffychat#1477]
### Feature Description

It would be a fantastic addition to Fluffychat's user-friendly interface to allow PDFs to be viewed directly within the client. When a PDF document is clicked, it should open seamlessly inside Fluffychat. This could be achieved using the following package:

https://pub.dev/packages/flutter_pdfview

### Rationale

Adding the ability to view PDFs directly within Fluffychat would enhance the user experience by providing a more seamless and integrated interaction.

**Comment by cortex3** [comment]
I agree, this is one of my biggest issues with fluffychat so far. Element X has this feature.

**Comment by biowilli** [comment]
The upstream maintainer decided against an in-app PDF renderer in #3050 

the added maintenance burden of a PDF rendering dependency isn't worth it
